### PR TITLE
First homework!

### PR DESCRIPTION
**Pull request by ghost**
Could you please check my home tasks.

Best regards. 
Yaroslav Gavrilovets

### CODE REVIEW

File: yGavrilovets/01_HTML/HW1_Python1.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<DOCTYPE html>
```
Comment (pluhin) [inline yGavrilovets/01_HTML/HW1_Python1.html]: You missed `!`

File: yGavrilovets/01_HTML/HW1_Python2.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<HTML>
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Python_2</Title>
+	</HEAD>
+	<BODY>
+		<H1>HOW-TO: Программа на Python, Часть 2</H1>
+		<UL>
+			<LI><em></em>Программа на Python — часть 1</LI>
```
Comment (pluhin) [inline yGavrilovets/01_HTML/HW1_Python2.html]: Something is wrong with `em` here :)

File: yGavrilovets/01_HTML/HW1_Python2.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+		<code>print len(months)</code>
+		<P>которая вернёт 12.</P>
+		<P>Ещё один пример списков — категории в поваренной книге. Например…</P>
+		<code>categories = ['Main dish','Meat','Fish','Soup','Cookies']</code>
+		<P>Таким образом, в <b>categories[0]</b> хранится <i>'Main dish'</i>, а в <b>categories[4]</b> — <i>'Cookies'</i>. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</P>
+		<HR>
+		<H2>Комментарии</H2>
+		<P>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак <em>'#'</em>.Например:</P><P>#</P><P>Это комментарий</P><P>Комментарии можно вставлять в любое место в строке, но помните, что Python проигнорирует всё, что идёт после <em>'#'</em>.</P>
```
Comment (pluhin) [inline yGavrilovets/01_HTML/HW1_Python2.html]:
```html
<P>#</P>
```
It is code section, not paragraph

File: yGavrilovets/01_HTML/HW1_additionalTask.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+		<H2>Теги над­ и подстрочных символов</H2>
+		<H3 align="center">Магазин работает:</H3>
+		<P align="center"center>c 08<sup><u>30</u></sup> до 18<sup><u>30</u></sup><br>обед с 13<sup><u>00</u></sup> до 14<sup><u>00</u></sup><br>выходной: <u>суббота</u>, <u>воскресенье</u></P>
+		<HR>
+		<H2>Вот такой текст</H2>
+		<H1><sup>Э</sup><sub>т</sub><sup>о</sup> <sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup><sub>т</sub><sup>о</sup> <sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub> <sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></H1>
+		<HR>
+		<H2>Акроним и курсив</H2>
+		<P>&Tau;&Phi;: Y(<i>x</i>)=e<sup><i>-i&pi;x</i></sup>+log<sub>x</sub>4</P>
+		<!--ЁБАНЫЕ ФООООРМУЛЫ!!!!!!!!-->
```
Comment (pluhin) [inline yGavrilovets/01_HTML/HW1_additionalTask.html]: Could you please remove this line

**Review by pluhin — CHANGES_REQUESTED** [review]

File: yGavrilovets/01_HTML/HW1_additionalTask.html (hunk 4 above)
Comment (ghost) [reply yGavrilovets/01_HTML/HW1_additionalTask.html]: Ohhh((( Sorry for this, i was sure, i've deleted all comments. it was for intermediate stage of homework. there won't be such situations any more

**Review by pluhin — APPROVED** [review]

**Pull request merged**